### PR TITLE
Feature create projects and files

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/controller/ProjectController.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+@RestController
+@RequestMapping("/api")
+public class ProjectController {
+
+    /**
+     * Endpoint for create a project in data base
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/controller/ProjectController.java]: In this JavaDoc  is missing @return

File: src/main/java/org/fundacion/jala/converter/controller/ProjectController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ *
+ * @author Juan Pablo Gonzales
+ */
+package org.fundacion.jala.converter.controller;
+
+import org.fundacion.jala.converter.models.FileSQL;
+import org.fundacion.jala.converter.models.Project;
+import org.fundacion.jala.converter.models.ProjectSQL;
+import org.fundacion.jala.converter.service.RunCommand;
+import org.springframework.web.bind.annotation.*;
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/controller/ProjectController.java]: Specify more than *

File: src/main/java/org/fundacion/jala/converter/controller/ProjectController.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        RunCommand runCommand = new RunCommand();
+        runCommand.run(command);
+        String pathProject = (System.getProperty("user.dir") + "\\" + projectName);
+        Project project = ProjectSQL.insertProjectData(projectName, pathProject, userId);
+        return project.getId();
+    }
+
+
+    /**
+     * Endpoint for create a project in data base
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/controller/ProjectController.java]: ing is missing

Comment (JRojas999) [reply src/main/java/org/fundacion/jala/converter/controller/ProjectController.java]: in "create"

File: src/main/java/org/fundacion/jala/converter/controller/ProjectController.java (hunk 1 above)
Comment (jpgonzales27) [reply src/main/java/org/fundacion/jala/converter/controller/ProjectController.java]: it's already corrected cris

**Review by cristianfchq — APPROVED** [review]

**Review by JRojas999 — APPROVED** [review]

**Pull request merged**